cmd/csaf_checker/main.go

```diff
@@ -39,6 +39,10 @@ type options struct {
 	Rate       *float64 `long:"rate" short:"r" description:"The average upper limit of https operations per second"`
 	Years      *uint    `long:"years" short:"y" description:"Number of years to look back from now" value-name:"YEARS"`
 
+	RemoteValidator        string   `long:"validator" description:"URL to validate documents remotely" value-name:"URL"`
+	RemoteValidatorCache   string   `long:"validatorcache" description:"FILE to cache remote validations" value-name:"FILE"`
+	RemoteValidatorPresets []string `long:"validatorpreset" description:"One or more presets to validate remotely"`
```

Comment:
Maybe document that the default preset is `mandatory`.

Reply:
@tschmidtb51 2a40ab6393d1bf445c51255598366f73cd454f0c let `validatorpreset`  defaults to `mandatory`.
This is displayed with `-h/--help` and is therefore mentioned in the docs.  
I am going to do the same in the sister PR #312 to let the downloader behave the same way.

BTW: I have added `[OPTIONS] domain...' to the usage output to match the downloader.

Reply:
Ok. I closed #321 in favor of this one here.